src/index.html

```diff
@@ -1,13 +1,55 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="AppleMonoblock"
+         class="card__photo"
+      >
+      <h2 class="card__title">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </h2>
+      <span class="card__code"> Код товара: 195434 </span>
+      <div class="card__box-flex card__box-flex--margin">
+        <div class="wrap">
+          <div class="wrap__stars"></div>
+          <div class="wrap__stars"></div>
+          <div class="wrap__stars"></div>
+          <div class="wrap__stars"></div>
+          <div class="wrap__stars"></div>
+        </div>
+        <div class="review">
```

Comment:
no need to create a separate block for 1 line of text. It's not a separate logical block that can be reused